extensions/mailer/runtime/src/main/java/io/quarkus/mailer/SentMail.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.mailer;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents a sent mail.
+ * Instances of this class are sent using CDI events.
+ *
+ * @param from the sender address
+ * @param to the list of recipients
+ * @param cc the list of CC recipients
+ * @param bcc the list of BCC recipients
+ * @param replyTo the list of reply-to addresses
+ * @param bounceAddress the bounce address
+ * @param subject the subject
+ * @param textBody the text body
+ * @param htmlBody the HTML body
+ * @param headers the headers
+ * @param attachments the attachments
+ */
+public record SentMail(String from,
```

Comment:
How about renaming to `MailSent` to facilitate finding it during the IDE auto-completion?

Reply:
Or maybe call it `Receipt` if you are feeling adventurous :)

Reply:
> It's not a Receipt - it does not guarantee that the mail was delivered; it was just sent.

I just wanted to find a common term that would illustrate that a Mail was sent. A receipt doesn't necessarily guarantee that it was delivered, just a proof that the package was dispatched (making an analogy to snail mail)